package.json

```diff
@@ -12,7 +12,7 @@
     "webpack:watch": "webpack --watch-stdin",
     "less:watch": "onchange 'site/assets/less/**/*.less' -- npm run less:build",
     "less:build": "lessc site/assets/less/main.less --clean-css site/static/site.css",
-    "build:hugo": "hugo --quiet -s site -d ../releases",
```

Comment:
Seems reasonable to me, although I know almost nothing about Hugo so I'll defer to @lidel to double check me.